### ISSUE TITLE
/history/{handle} is not accessible without clicking/tapping

### ISSUE THREAD

**Issue by SiravitPhokeed** [Emocial-NLP-Depression-Detection/Emocial-Frontend#6]
**Seen in**: /history
**Affects**: All devices
**Intended behaviour**: All /history rows should be selectable by using the tab key
